Documentation/nvme-write.html

```diff
@@ -992,9 +992,9 @@ <h2 id="_options">OPTIONS</h2>
 </dt>
 <dd>
 <p>
-        The optional data set management attributes for this command. The
-        argument for this is the lower 16 bits of the DSM field in a write
-        command; the upper 16 bits of the field come from the directive
+        The optional data set management attributes for this command. The argument

```

Comment:
Also here, just update the source and remove the generated changes from the commit. I am going to regenerated all the precompiled documentation right before the release.

Reply:
removed :)